File: Estrutura de Dados Simples/Desafio.cpp
#include <iostream>
using namespace std;

int main() {
    string nomes[5];
    double precos[5];
    int quantidades[5];

    for (int i = 0; i < 5; i++) {
        cout << "\nProduto " << i + 1 << ":" << endl;

        cout << "Nome: ";
        cin >> nomes[i];

        cout << "Preço: ";
        cin >> precos[i];

        cout << "Quantidade: ";
        cin >> quantidades[i];
    }

    cout << "\n--- Lista de Produtos ---" << endl;
    for (int i = 0; i < 5; i++) {
        cout << nomes[i] << " - Preço: " << precos[i]
             << " - Quantidade: " << quantidades[i] << endl;
    }

    double total = 0;
    for (int i = 0; i < 5; i++) {
        total += precos[i] * quantidades[i];
    }

    cout << "\nValor total em estoque: " << total << endl;

    int indice = 0;
    for (int i = 1; i < 5; i++) {
        if (quantidades[i] > quantidades[indice]) {
            indice = i;
        }
    }

    cout << "Produto com maior quantidade: " << nomes[indice] << endl;

    return 0;
}

File: Estrutura de Dados Simples/Desafio.py
nomes = []
precos = []
quantidades = []

for i in range(5):
    print(f"\nProduto {i+1}:")
    nomes.append(input("Nome: "))
    precos.append(float(input("Preço: ")))
    quantidades.append(int(input("Quantidade: ")))

# Mostrar produtos
print("\n--- Lista de Produtos ---")
for i in range(5):
    print(f"{nomes[i]} - Preço: {precos[i]} - Quantidade: {quantidades[i]}")

# Calcular valor total
total = 0
for i in range(5):
    total += precos[i] * quantidades[i]

print("\nValor total em estoque:", total)

# Produto com maior quantidade
maior = quantidades[0]
indice = 0

for i in range(1, 5):
    if quantidades[i] > maior:
        maior = quantidades[i]
        indice = i

print("Produto com maior quantidade:", nomes[indice])